kessler/kessler.meta

```diff
@@ -168,7 +168,7 @@
   type = integer
   intent = in
 [ rho ]
-  standard_name = density_of_dry_air_at_stp
+  standard_name = density_of_dry_air
```

Comment:
Following the Standard Names dictionary (https://github.com/ESCOMP/CCPPStandardNames/blob/main/Metadata-standard-names.md), this should probably be: `dry_air_density`

Reply:
Good call!  Name changed.